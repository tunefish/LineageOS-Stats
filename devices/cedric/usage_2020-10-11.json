{
  "1.0.1-cedric": 1,
  "14.1-20170601-unofficial-cedric": 1,
  "14.1-20170706-unofficial-cedric": 3,
  "14.1-20170811-unofficial-cedric": 20,
  "14.1-20171006-unofficial-cedric": 3,
  "14.1-20171018-unofficial-cedric": 4,
  "14.1-20171111_140556-unofficial-cedric": 18,
  "14.1-20180117-unofficial-cedric": 23,
  "14.1-20180318-unofficial-cedric": 39,
  "14.1-20180817-unofficial-cedric": 38,
  "14.1-20190207-unofficial-cedric": 1,
  "14.1-20190221-unofficial-cedric": 2,
  "14.1-20190311-unofficial-cedric": 1,
  "14.1-20190407-unofficial-cedric": 2,
  "14.1-20190630-unofficial-cedric": 4,
  "14.1-20190706-unofficial-cedric": 1,
  "14.1-20190710-unofficial-cedric": 3,
  "14.1-20190813-unofficial-cedric": 4,
  "14.1-20190912-unofficial-cedric": 18,
  "14.1-20191115-unofficial-cedric": 9,
  "14.1-20191228-unofficial-cedric": 1,
  "14.1-20200214-unofficial-cedric": 1,
  "14.1-20200216-unofficial-cedric": 1,
  "14.1-20200304-unofficial-cedric": 1,
  "14.1-20200328-unofficial-cedric": 1,
  "14.1-20200407-unofficial-cedric": 1,
  "14.1-20200424-unofficial-cedric": 3,
  "14.1-20200508-unofficial-cedric": 2,
  "14.1-20200610-unofficial-cedric": 3,
  "14.1-20200805-unofficial-cedric": 3,
  "14.1-20200822-unofficial-cedric": 6,
  "14.1-20200910-unofficial-cedric": 1,
  "14.1-20200919-unofficial-cedric": 12,
  "14.1-20201006-unofficial-cedric": 2,
  "15.1-20180213-unofficial-cedric": 20,
  "15.1-20180415-unofficial-cedric": 83,
  "15.1-20180422-unofficial-cedric": 5,
  "15.1-20180424-unofficial-cedric": 1,
  "15.1-20180723-unofficial-cedric": 2,
  "15.1-20180828-unofficial-cedric": 2,
  "15.1-20190425-unofficial-cedric": 3,
  "15.1-20200927-unofficial-cedric": 3,
  "15.1-20201003-unofficial-cedric": 9,
  "16.0-20190101-unofficial-cedric": 39,
  "16.0-20190405-unofficial-cedric": 20,
  "16.0-20190522-unofficial-cedric": 23,
  "16.0-20190525-unofficial-cedric": 1,
  "16.0-20190701-unofficial-cedric": 2,
  "16.0-20190709-unofficial-cedric": 37,
  "16.0-20190711-unofficial-cedric": 2,
  "16.0-20190719-unofficial-cedric": 1,
  "16.0-20190817-unofficial-cedric": 1,
  "16.0-20190912-unofficial-cedric": 153,
  "16.0-20191016-unofficial-cedric": 1,
  "16.0-20191211-unofficial-cedric": 1,
  "16.0-20191214-unofficial-cedric": 2,
  "16.0-20200211-unofficial-cedric": 1,
  "16.0-20200311-unofficial-cedric": 2,
  "16.0-20200511-unofficial-cedric": 1,
  "16.0-20200522-unofficial-cedric": 1,
  "16.0-20200819-unofficial-cedric": 1,
  "16.0-20200914-unofficial-cedric": 1,
  "16.0-20200915-unofficial-cedric": 1,
  "16.0-20200919-unofficial-cedric": 2,
  "17.0-20191214-unofficial-cedric": 109,
  "17.1-20200112-unofficial-cedric": 18,
  "17.1-20200220-unofficial-cedric": 32,
  "17.1-20200319-unofficial-cedric": 3,
  "17.1-20200419-unofficial-cedric": 1,
  "17.1-20200422-unofficial-cedric": 90,
  "17.1-20200520-unofficial-cedric": 2,
  "17.1-20200602-unofficial-cedric": 1,
  "17.1-20200612-unofficial-cedric": 148,
  "17.1-20200623-unofficial-cedric": 1,
  "17.1-20200625-unofficial-cedric": 1,
  "17.1-20200709-unofficial-cedric": 1,
  "17.1-20200715-unofficial-cedric": 2,
  "17.1-20200717-unofficial-cedric": 1,
  "17.1-20200724-unofficial-cedric": 1,
  "17.1-20200726-unofficial-cedric": 1,
  "17.1-20200731-unofficial-cedric": 1,
  "17.1-20200808-unofficial-cedric": 1,
  "17.1-20200810-unofficial-cedric": 4,
  "17.1-20200813-unofficial-cedric": 2,
  "17.1-20200814-unofficial-cedric": 1,
  "17.1-20200815-unofficial-cedric": 2,
  "17.1-20200816-unofficial-cedric": 1,
  "17.1-20200818-unofficial-cedric": 2,
  "17.1-20200819-unofficial-cedric": 1,
  "17.1-20200820-unofficial-cedric": 369,
  "17.1-20200822-unofficial-cedric": 62,
  "17.1-20200905-unofficial-cedric": 1,
  "17.1-20200908-unofficial-cedric": 1,
  "17.1-20200909-unofficial-cedric": 1,
  "17.1-20200910-unofficial-cedric": 1,
  "17.1-20200911-unofficial-cedric": 2,
  "17.1-20200913-unofficial-cedric": 2,
  "17.1-20200914-unofficial-cedric": 1,
  "17.1-20200915-unofficial-cedric": 5,
  "17.1-20200917-unofficial-cedric": 1,
  "17.1-20200919-unofficial-cedric": 1,
  "17.1-20200920-unofficial-cedric": 5,
  "17.1-20200921-unofficial-cedric": 4,
  "17.1-20200922-unofficial-cedric": 43,
  "17.1-20200927-unofficial-cedric": 1,
  "17.1-20201004-unofficial-cedric": 5,
  "17.1-20201006-unofficial-cedric": 2,
  "17.1-20201009-unofficial-cedric": 1,
  "aosp_cedric-userdebug 9 pq3a.190801.002 eng.thxbro.20190813.181708 test-keys": 1,
  "qq3a.200705.002": 8,
  "ro.lineage.releasetype=unofficial": 3,
  "rr_cedric-userdebug 10 qp1a.191105.004 eng.root.20191215.073423 test-keys": 6
}