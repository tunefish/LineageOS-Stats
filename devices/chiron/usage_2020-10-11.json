{
  "0.11-q-2020092775558-test-chiron": 1,
  "10.0-20200628-unofficial-snapshot-chiron": 3,
  "10.0-20200720-unofficial-snapshot-chiron": 1,
  "10.0-20200820-unofficial-snapshot-chiron": 3,
  "10.0-20200913-unofficial-snapshot-chiron": 33,
  "14.1-20171012-unofficial-chiron": 3,
  "14.1-20171123_135319-unofficial-chiron": 1,
  "14.1-20180215-unofficial-chiron": 1,
  "14.1-20200104-unofficial-chiron": 27,
  "15.1-20180507-nightly-chiron": 1,
  "15.1-20180521-nightly-chiron": 3,
  "15.1-20180605-nightly-chiron": 1,
  "15.1-20180618-nightly-chiron": 1,
  "15.1-20180625-nightly-chiron": 2,
  "15.1-20180702-nightly-chiron": 3,
  "15.1-20180709-nightly-chiron": 3,
  "15.1-20180716-nightly-chiron": 1,
  "15.1-20180723-nightly-chiron": 2,
  "15.1-20180806-nightly-chiron": 2,
  "15.1-20180813-nightly-chiron": 1,
  "15.1-20180820-nightly-chiron": 2,
  "15.1-20180828-unofficial-chiron": 2,
  "15.1-20180910-nightly-chiron": 3,
  "15.1-20181001-nightly-chiron": 2,
  "15.1-20181008-nightly-chiron": 2,
  "15.1-20181128-nightly-chiron": 1,
  "15.1-20181204-nightly-chiron": 1,
  "15.1-20181207-nightly-chiron": 1,
  "15.1-20181214-nightly-chiron": 1,
  "15.1-20181220-nightly-chiron": 1,
  "15.1-20181229-nightly-chiron": 1,
  "15.1-20181230-nightly-chiron": 1,
  "15.1-20190104-nightly-chiron": 1,
  "15.1-20190106-nightly-chiron": 1,
  "15.1-20190107-nightly-chiron": 1,
  "15.1-20190122-nightly-chiron": 1,
  "15.1-20190124-nightly-chiron": 1,
  "15.1-20190127-nightly-chiron": 1,
  "15.1-20190130-nightly-chiron": 1,
  "15.1-20190203-nightly-chiron": 1,
  "15.1-20190214-nightly-chiron": 1,
  "15.1-20190223-nightly-chiron": 1,
  "15.1-20190224-nightly-chiron": 1,
  "15.1-20190305-nightly-chiron": 2,
  "15.1-20190312-nightly-chiron": 1,
  "15.1-20190326-nightly-chiron": 3,
  "15.1-20190402-nightly-chiron": 3,
  "15.1-20190409-nightly-chiron": 1,
  "15.1-20190416-nightly-chiron": 1,
  "15.1-20190418-unofficial-chiron": 3,
  "15.1-20190423-nightly-chiron": 2,
  "15.1-20190430-nightly-chiron": 1,
  "15.1-20190507-nightly-chiron": 2,
  "15.1-20190514-nightly-chiron": 2,
  "15.1-20190521-nightly-chiron": 2,
  "15.1-20190528-nightly-chiron": 2,
  "15.1-20190604-nightly-chiron": 27,
  "15.1-20191013-unofficial-mido": 3,
  "15.1-20200113-unofficial-chiron": 14,
  "16.0-20181016-unofficial-chiron": 2,
  "16.0-20181022-unofficial-chiron": 1,
  "16.0-20181202-unofficial-chiron": 1,
  "16.0-20190210-unofficial-chiron": 1,
  "16.0-20190228-unofficial-chiron": 2,
  "16.0-20190309-unofficial-chiron": 1,
  "16.0-20190319-unofficial-chiron": 1,
  "16.0-20190409-unofficial-chiron": 3,
  "16.0-20190609-unofficial-chiron": 1,
  "16.0-20190614-nightly-chiron": 1,
  "16.0-20190619-nightly-chiron": 1,
  "16.0-20190621-nightly-chiron": 1,
  "16.0-20190626-nightly-chiron": 1,
  "16.0-20190627-nightly-chiron": 2,
  "16.0-20190629-nightly-chiron": 1,
  "16.0-20190703-nightly-chiron": 1,
  "16.0-20190712-nightly-chiron": 2,
  "16.0-20190713-unofficial-chiron": 1,
  "16.0-20190714-nightly-chiron": 2,
  "16.0-20190718-nightly-chiron": 1,
  "16.0-20190719-nightly-chiron": 2,
  "16.0-20190722-nightly-chiron": 1,
  "16.0-20190728-nightly-chiron": 1,
  "16.0-20190729-nightly-chiron": 1,
  "16.0-20190730-nightly-chiron": 1,
  "16.0-20190802-nightly-chiron": 1,
  "16.0-20190803-nightly-chiron": 2,
  "16.0-20190804-nightly-chiron": 2,
  "16.0-20190806-nightly-chiron": 2,
  "16.0-20190808-nightly-chiron": 1,
  "16.0-20190809-nightly-chiron": 4,
  "16.0-20190817-unofficial-chiron": 3,
  "16.0-20190825-unofficial-chiron": 1,
  "16.0-20190924-nightly-chiron": 1,
  "16.0-20191006-nightly-chiron": 1,
  "16.0-20191017-nightly-chiron": 1,
  "16.0-20191023-nightly-chiron": 2,
  "16.0-20191027-nightly-chiron": 1,
  "16.0-20191029-nightly-chiron": 1,
  "16.0-20191103-nightly-chiron": 1,
  "16.0-20191104-nightly-chiron": 1,
  "16.0-20191105-nightly-chiron": 1,
  "16.0-20191115-nightly-chiron": 1,
  "16.0-20191124-nightly-chiron": 1,
  "16.0-20191207-nightly-chiron": 1,
  "16.0-20191208-nightly-chiron": 3,
  "16.0-20191212-nightly-chiron": 1,
  "16.0-20191220-nightly-chiron": 2,
  "16.0-20191225-nightly-chiron": 1,
  "16.0-20191227-nightly-chiron": 1,
  "16.0-20191228-nightly-chiron": 1,
  "16.0-20200106-nightly-chiron": 1,
  "16.0-20200111-nightly-chiron": 1,
  "16.0-20200114-nightly-chiron": 1,
  "16.0-20200115-nightly-chiron": 1,
  "16.0-20200116-nightly-chiron": 1,
  "16.0-20200120-nightly-chiron": 1,
  "16.0-20200121-nightly-chiron": 1,
  "16.0-20200122-nightly-chiron": 1,
  "16.0-20200128-nightly-chiron": 3,
  "16.0-20200129-nightly-chiron": 1,
  "16.0-20200202-nightly-chiron": 1,
  "16.0-20200203-nightly-chiron": 1,
  "16.0-20200206-nightly-chiron": 1,
  "16.0-20200209-nightly-chiron": 1,
  "16.0-20200212-nightly-chiron": 1,
  "16.0-20200220-nightly-chiron": 1,
  "16.0-20200222-nightly-chiron": 2,
  "16.0-20200223-nightly-chiron": 2,
  "16.0-20200224-nightly-chiron": 3,
  "16.0-20200301-nightly-chiron": 1,
  "16.0-20200302-nightly-chiron": 1,
  "16.0-20200303-nightly-chiron": 4,
  "16.0-20200304-nightly-chiron": 3,
  "16.0-20200305-nightly-chiron": 3,
  "16.0-20200307-nightly-chiron": 2,
  "16.0-20200309-microg-chiron": 1,
  "16.0-20200309-nightly-chiron": 1,
  "16.0-20200310-nightly-chiron": 2,
  "16.0-20200311-nightly-chiron": 1,
  "16.0-20200312-nightly-chiron": 2,
  "16.0-20200313-nightly-chiron": 1,
  "16.0-20200314-nightly-chiron": 3,
  "16.0-20200315-nightly-chiron": 1,
  "16.0-20200321-nightly-chiron": 1,
  "16.0-20200322-nightly-chiron": 1,
  "16.0-20200323-nightly-chiron": 3,
  "16.0-20200325-nightly-chiron": 48,
  "16.0-20200326-nightly-chiron": 1,
  "16.0-20200331-microg-chiron": 1,
  "17.1-20200111-unofficial-chiron": 22,
  "17.1-20200404-nightly-chiron": 1,
  "17.1-20200405-nightly-chiron": 1,
  "17.1-20200406-nightly-chiron": 1,
  "17.1-20200415-nightly-chiron": 1,
  "17.1-20200422-nightly-chiron": 11,
  "17.1-20200424-nightly-chiron": 1,
  "17.1-20200426-nightly-chiron": 1,
  "17.1-20200429-nightly-chiron": 1,
  "17.1-20200530-nightly-chiron": 1,
  "17.1-20200531-nightly-chiron": 7,
  "17.1-20200602-nightly-chiron": 2,
  "17.1-20200604-nightly-chiron": 6,
  "17.1-20200611-nightly-chiron": 6,
  "17.1-20200618-nightly-chiron": 7,
  "17.1-20200625-nightly-chiron": 7,
  "17.1-20200702-nightly-chiron": 7,
  "17.1-20200709-nightly-chiron": 6,
  "17.1-20200716-nightly-chiron": 5,
  "17.1-20200723-nightly-chiron": 6,
  "17.1-20200730-nightly-chiron": 10,
  "17.1-20200806-nightly-chiron": 9,
  "17.1-20200813-nightly-chiron": 15,
  "17.1-20200818-microg-chiron": 3,
  "17.1-20200823-unofficial-chiron": 1,
  "17.1-20200827-nightly-chiron": 15,
  "17.1-20200903-nightly-chiron": 34,
  "17.1-20200909-unofficial-chiron": 1,
  "17.1-20200910-nightly-chiron": 178,
  "17.1-20200917-nightly-chiron": 4,
  "17.1-20200922-microg-chiron": 3,
  "17.1-20200924-nightly-chiron": 29,
  "17.1-20200930-unofficial-chiron": 1,
  "17.1-20201001-nightly-chiron": 179,
  "17.1-20201002-unofficial-chiron": 1,
  "17.1-20201007-unofficial-chiron": 1,
  "17.1-20201008-nightly-chiron": 116,
  "18.0-20201005-unofficial-chiron": 38,
  "18.0-20201008-unofficial-chiron": 15,
  "chiron-user 7.1.2opr1.170623.027 04xb4xl1af release-keys": 1,
  "chiron-user 7.1.2opr1.170623.027 1817cn9q8m release-keys": 1,
  "chiron-user 7.1.2opr1.170623.027 1u3g7j7wwv release-keys": 1,
  "chiron-user 7.1.2opr1.170623.027 1w1y5l331v release-keys": 1,
  "chiron-user 7.1.2opr1.170623.027 231zzm9c76 release-keys": 1,
  "chiron-user 7.1.2opr1.170623.027 28o3h8122m release-keys": 1,
  "chiron-user 7.1.2opr1.170623.027 28p3ps15rt release-keys": 1,
  "chiron-user 7.1.2opr1.170623.027 329elam0s6 release-keys": 1,
  "chiron-user 7.1.2opr1.170623.027 3g4gy9i37e release-keys": 1,
  "chiron-user 7.1.2opr1.170623.027 48af2r3941 release-keys": 1,
  "chiron-user 7.1.2opr1.170623.027 65hz3day01 release-keys": 1,
  "chiron-user 7.1.2opr1.170623.027 74gtm26v02 release-keys": 1,
  "chiron-user 7.1.2opr1.170623.027 75z103ddht release-keys": 1,
  "chiron-user 7.1.2opr1.170623.027 88403p0ks9 release-keys": 1,
  "chiron-user 7.1.2opr1.170623.027 8980784si7 release-keys": 1,
  "chiron-user 7.1.2opr1.170623.027 912c795mv8 release-keys": 1,
  "chiron-user 7.1.2opr1.170623.027 95xbgxo5a1 release-keys": 1,
  "chiron-user 7.1.2opr1.170623.027 ax3u11w6d6 release-keys": 1,
  "chiron-user 7.1.2opr1.170623.027 eewtfwi39t release-keys": 1,
  "chiron-user 7.1.2opr1.170623.027 g2fao36pq6 release-keys": 1,
  "chiron-user 7.1.2opr1.170623.027 i9v3yianbh release-keys": 1,
  "chiron-user 7.1.2opr1.170623.027 ko1k4o5rw2 release-keys": 1,
  "chiron-user 7.1.2opr1.170623.027 l6rdk8166b release-keys": 1,
  "chiron-user 7.1.2opr1.170623.027 mpl5v1304f release-keys": 1,
  "chiron-user 7.1.2opr1.170623.027 pf10h1cb6e release-keys": 1,
  "chiron-user 7.1.2opr1.170623.027 rhj4u2t771 release-keys": 1,
  "chiron-user 7.1.2opr1.170623.027 uk717dj2s1 release-keys": 1,
  "chiron-user 7.1.2opr1.170623.027 uyogpwqg5t release-keys": 1,
  "chiron-user 7.1.2opr1.170623.027 vz861evm23 release-keys": 1,
  "chiron-user 7.1.2opr1.170623.027 w86d0iy4e3 release-keys": 1,
  "chiron-user 7.1.2opr1.170623.027 yuwm3xh7ot release-keys": 1,
  "chiron-user 7.1.2opr1.170623.027 z41vgiv79u release-keys": 1,
  "lineage_gemini-userdebug 7.1.2 njh47f 8290c49eea test-keys": 13,
  "mi mix 2": 16,
  "msm8953_64-user 7.1.2honorbkk-al10 7v2pr4y8a8 release-keys": 1
}