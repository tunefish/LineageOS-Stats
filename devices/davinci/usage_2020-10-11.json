{
  "0.11-q-20200921-unofficial-davinci": 20,
  "0.12-q-20201007-unofficial-davinci": 1,
  "10.0-20200921-unofficial-snapshot-davinci": 1,
  "10.0-20200926-unofficial-snapshot-davinci": 4,
  "10.0-20201009-unofficial-snapshot-davinci": 2,
  "16.0-20190824-unofficial-davinci": 4,
  "16.0-20190921-unofficial-davinci": 21,
  "16.0-20200525-unofficial-hammerhead": 2,
  "17.0-20191231-unofficial-pig-gapps-davinci": 5,
  "17.0-20200101-unofficial-pig-davinci": 2,
  "17.1-20200202-unofficial-pig-davinci": 5,
  "17.1-20200310-unofficial-pig-davinci": 3,
  "17.1-20200327-unofficial-pig-davinci": 3,
  "17.1-20200404-unofficial-pig-davinci": 1,
  "17.1-20200405-unofficial-pig-davinci": 1,
  "17.1-20200410-unofficial-pig-davinci": 7,
  "17.1-20200419-unofficial-pig-davinci": 7,
  "17.1-20200429-unofficial-davinci": 1,
  "17.1-20200502-unofficial-davinci": 9,
  "17.1-20200503-unofficial-davinci": 2,
  "17.1-20200608-unofficial-pig-davinci": 38,
  "17.1-20200720-unofficial-pig-davinci": 2,
  "17.1-20200722-unofficial-pig-davinci": 86,
  "17.1-20200810-unofficial-davinci": 1,
  "17.1-20200814-unofficial-davinci": 1,
  "17.1-20200817-unofficial-davinci": 1,
  "17.1-20200818-unofficial-davinci": 1,
  "17.1-20200823-unofficial-davinci": 1,
  "17.1-20200824-unofficial-gms-davinci": 1,
  "17.1-20200829-unofficial-davinci": 1,
  "17.1-20200830-unofficial-davinci": 3,
  "17.1-20200830-unofficial-gms-davinci": 1,
  "17.1-20200906-unofficial-davinci": 2,
  "17.1-20200906-unofficial-gms-davinci": 1,
  "17.1-20200906-unofficial-pig-davinci": 1,
  "17.1-20200913-davinci": 1,
  "17.1-20200913-unofficial-davinci": 2,
  "17.1-20200913-unofficial-gms-davinci": 15,
  "17.1-20200920-unofficial-davinci": 34,
  "17.1-20200921-unofficial-davinci": 1,
  "17.1-20200921-unofficial-gms-davinci": 1,
  "17.1-20200922-unofficial-davinci": 21,
  "17.1-20200922-unofficial-gms-davinci": 18,
  "17.1-20200924-unofficial-davinci": 1,
  "17.1-20200926-unofficial-pig-eol-davinci": 136,
  "17.1-20200927-unofficial-davinci": 39,
  "17.1-20200927-unofficial-gms-davinci": 42,
  "17.1-20200929-unofficial-davinci": 1,
  "17.1-20201004-unofficial-davinci": 27,
  "17.1-20201004-unofficial-gms-davinci": 5,
  "17.1-20201005-unofficial-gms-davinci": 25,
  "17.1-20201007-nightly-davinci": 345,
  "17.1-20201009-unofficial-davinci": 2,
  "18.0-20201010-unofficial-davinci": 1,
  "5.1-20200705-sana_public-davinci": 2,
  "5.1-20200921-sana_public-davinci": 1,
  "davinci-user 7.1.2pkq1.190302.001 063dvhsnrm release-keys": 1,
  "davinci-user 7.1.2pkq1.190302.001 0b936h9yj9 release-keys": 1,
  "davinci-user 7.1.2pkq1.190302.001 0vd003s2xv release-keys": 1,
  "davinci-user 7.1.2pkq1.190302.001 0x4ywvto1s release-keys": 1,
  "davinci-user 7.1.2pkq1.190302.001 133j2fo2ye release-keys": 1,
  "davinci-user 7.1.2pkq1.190302.001 2389p5m64x release-keys": 1,
  "davinci-user 7.1.2pkq1.190302.001 23yoxrb09x release-keys": 1,
  "davinci-user 7.1.2pkq1.190302.001 2s720qd943 release-keys": 1,
  "davinci-user 7.1.2pkq1.190302.001 3t2m1z8br6 release-keys": 1,
  "davinci-user 7.1.2pkq1.190302.001 44p9g2t34k release-keys": 1,
  "davinci-user 7.1.2pkq1.190302.001 4d1ibc6qdn release-keys": 1,
  "davinci-user 7.1.2pkq1.190302.001 58tds6huio release-keys": 1,
  "davinci-user 7.1.2pkq1.190302.001 5r23t8ie98 release-keys": 1,
  "davinci-user 7.1.2pkq1.190302.001 63m06fecc7 release-keys": 1,
  "davinci-user 7.1.2pkq1.190302.001 6593d8z825 release-keys": 1,
  "davinci-user 7.1.2pkq1.190302.001 6c606cn2g0 release-keys": 1,
  "davinci-user 7.1.2pkq1.190302.001 6p71x4i3y8 release-keys": 1,
  "davinci-user 7.1.2pkq1.190302.001 74841790hc release-keys": 1,
  "davinci-user 7.1.2pkq1.190302.001 7ekcq487cv release-keys": 1,
  "davinci-user 7.1.2pkq1.190302.001 7m8te99rxw release-keys": 1,
  "davinci-user 7.1.2pkq1.190302.001 80de6fzf27 release-keys": 1,
  "davinci-user 7.1.2pkq1.190302.001 8mo94jc297 release-keys": 1,
  "davinci-user 7.1.2pkq1.190302.001 9d31v9a5aq release-keys": 1,
  "davinci-user 7.1.2pkq1.190302.001 9h358l60y3 release-keys": 1,
  "davinci-user 7.1.2pkq1.190302.001 cg967zgmx3 release-keys": 1,
  "davinci-user 7.1.2pkq1.190302.001 cv4qc338aq release-keys": 1,
  "davinci-user 7.1.2pkq1.190302.001 fng94k5d1o release-keys": 1,
  "davinci-user 7.1.2pkq1.190302.001 hb7910851b release-keys": 1,
  "davinci-user 7.1.2pkq1.190302.001 i4cd887421 release-keys": 1,
  "davinci-user 7.1.2pkq1.190302.001 j8o9x2j64i release-keys": 1,
  "davinci-user 7.1.2pkq1.190302.001 jb2n5303dj release-keys": 1,
  "davinci-user 7.1.2pkq1.190302.001 jxegrdf6y5 release-keys": 1,
  "davinci-user 7.1.2pkq1.190302.001 k7rvgr57e8 release-keys": 1,
  "davinci-user 7.1.2pkq1.190302.001 kb7554f67r release-keys": 1,
  "davinci-user 7.1.2pkq1.190302.001 krt8503uvn release-keys": 1,
  "davinci-user 7.1.2pkq1.190302.001 nd60ns1a8w release-keys": 1,
  "davinci-user 7.1.2pkq1.190302.001 omw63290kh release-keys": 1,
  "davinci-user 7.1.2pkq1.190302.001 p5do4vc78n release-keys": 1,
  "davinci-user 7.1.2pkq1.190302.001 qa7s04321i release-keys": 1,
  "davinci-user 7.1.2pkq1.190302.001 r25vd72y47 release-keys": 1,
  "davinci-user 7.1.2pkq1.190302.001 sl191aac0t release-keys": 1,
  "davinci-user 7.1.2pkq1.190302.001 urj7557mh4 release-keys": 1,
  "davinci-user 7.1.2pkq1.190302.001 urw3vp958x release-keys": 1,
  "davinci-user 7.1.2pkq1.190302.001 us6vdetc21 release-keys": 1,
  "davinci-user 7.1.2pkq1.190302.001 wpg3ngw50m release-keys": 1,
  "davinci-user 7.1.2pkq1.190302.001 x433391h8m release-keys": 1,
  "davinci-user 7.1.2pkq1.190302.001 yz32561d80 release-keys": 1,
  "mi 9t": 13,
  "qq3a.200805.001": 36,
  "qq3a.200905.001 test-keys": 1,
  "rr_ginkgo-userdebug 10 qq3a.200805.001 eng.dqrkn3.20200826.170409 test-keys": 2,
  "rr_ginkgo-userdebug 10 qq3a.200905.001 eng.dqrkn3.20200913.175106 release-keys": 2,
  "rr_ginkgo-userdebug 10 qq3a.200905.001 eng.dqrkn3.20200918.060731 release-keys": 3,
  "rr_ginkgo-userdebug 10 qq3a.200905.001 eng.dqrkn3.20201002.140152 release-keys": 2,
  "rr_j6primelte-userdebug 10 qq3a.200905.001 eng.msm891.20201004.024114 release-keys": 1,
  "rr_onclite-userdebug 10 qq3a.200805.001 eng.ubuntu.20200815.151955 test-keys": 1,
  "rr_onclite-userdebug 10 qq3a.200805.001 eng.ubuntu.20200909.140723 release-keys": 2,
  "rr_twolip-userdebug 10 qq3a.200905.001 3 release-keys": 1,
  "rr_twolip-userdebug 10 qq3a.200905.001 4 release-keys": 1,
  "rr_vince-userdebug 10 qq3a.200905.001 eng.rutura.20200929.134313 release-keys": 1,
  "rr_x00t-userdebug 10 qq3a.200905.001 eng.sonicb.20201001.225014 test-keys": 1,
  "rr_ysl-userdebug 10 qq3a.200805.001 eng.tushar.20200818.124919 test-keys": 1,
  "ugg-user 7.1.2n2g47h cc50w3lty1 release-keys": 1
}