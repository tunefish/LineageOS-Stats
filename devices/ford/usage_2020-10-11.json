{
  "12.1-20180130-unofficial-ford": 243,
  "12.1-20180602-unofficial-ford": 22,
  "12.1-20180818-unofficial-ford": 1,
  "12.1-20180912-unofficial-ford": 1,
  "12.1-20181109-unofficial-ford": 2,
  "12.1-20181218-unofficial-ford": 826,
  "12.1-20190422-unofficial-ford": 5,
  "12.1-20190524-unofficial-ford": 10,
  "12.1-20190731-unofficial-ford": 16,
  "12.1-20190823-unofficial-ford": 3,
  "12.1-20200423-unofficial-ford": 1,
  "12.1-20200505-unofficial-ford": 38,
  "14.1-20190828-unofficial-ford": 211
}