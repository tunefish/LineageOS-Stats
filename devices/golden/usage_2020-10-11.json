{
  "13.0-20170129-unofficial-golden": 35,
  "13.0-20170224-unofficial-golden": 5,
  "13.0-20170302-unofficial-golden": 4,
  "13.0-20170310-unofficial-golden": 10,
  "13.0-20170328-unofficial-golden": 212,
  "13.0-20170420-unofficial-golden": 14,
  "14.1-20170122-unofficial-golden": 111,
  "14.1-20170318-unofficial-golden": 2,
  "14.1-20170402-unofficial-golden": 1,
  "14.1-20170415-unofficial-golden": 3,
  "14.1-20170419-unofficial-golden": 2,
  "14.1-20170507-unofficial-golden": 11,
  "14.1-20170522-unofficial-golden": 1,
  "14.1-20170609-unofficial-golden": 192,
  "14.1-20170610-unofficial-golden": 4,
  "14.1-20170628-unofficial-golden": 346,
  "14.1-20170724-unofficial-golden": 19,
  "14.1-20170817-unofficial-golden": 13,
  "14.1-20170920-unofficial-golden": 3,
  "14.1-20171011-unofficial-golden": 23,
  "14.1-20171215_192311-official-golden": 28,
  "14.1-20171215_192311-unofficial-golden": 269,
  "14.1-20180109_002536-unofficial-golden": 68,
  "14.1-20181014-unofficial-golden": 1,
  "14.1-20200817_071197-official-golden-rogerwed": 1,
  "15.1-20181030-unofficial-golden": 2,
  "golden-user 8.1.0 oe632e 3de271a2ce release-keys": 1
}